rental-car-manager/editorextensions/rental-car-manager/src/rules/color.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ConditionType,
+  ShapeDataConditions,
+  ConditionCombination,
+  RuleFormattingType,
+  DataGraphicIconSets,
+  HorizontalBadgePos,
+  VerticalBadgePos,
+  BadgeLayerPos,
+  BadgeResponsiveness,
+} from "lucid-extension-sdk";
+import { RuleDefinition } from "lucid-extension-sdk/document/ruledefinition";
+
+export const carColorRuleDefinition: RuleDefinition = {
+  name: "Car color icon",
+  effects: [
+    {
+      conditions: [
+        {
+          type: ConditionType.ShapeData,
+          condition: ShapeDataConditions.Equal,
+          value: "Red",
+          field: "Color Name",
+        },
+      ],
+      combination: ConditionCombination.AND,
```

Comment:
do we need to specify a combination here? is it optional?

Admittedly, I don't really understand how to specify conditional formatting (I can go look it up now if that's helpful), so take what I say with a grain of salt